docs/options.md

```diff
@@ -21,11 +21,40 @@ or `axios` section in `nuxt.config.js`:
 }
 ```
 
+### Environment Variables
```

Comment:
Since source of runtimeConfig is not necessary environment variables and only  baseURL* can be configured, what do you think renaming section `RuntimeConfig` or `Runtime BaseURL` or `Dynamic BaseURL`?

Reply:
Should I also move it to `baseURL` section?

Reply:
I think here is good if title is runtimeConfig. Maybe other options will be introduced to for runtime support.